manifests/init.pp

```diff
@@ -243,7 +247,10 @@
   $ssl_port                                      = $rabbitmq::params::ssl_port,
   Optional[String] $ssl_interface                = undef,
   Integer $ssl_management_port                   = $rabbitmq::params::ssl_management_port,
-  Integer $ssl_stomp_port                                = $rabbitmq::params::ssl_stomp_port,
+  Variant[Undef, Boolean, String] $ssl_management_cacert = $ssl_cacert,
```

Comment:
I think there's some issue trying to default one parameter based on another like this.  @bastelfreak this came up on IRC IIRC??

Reply:
Actually, since puppet 4.4.0 (I've been go back through the IRC logs), this should work. https://github.com/puppetlabs/puppet-specifications/blob/master/language/parameter_scope.md#the-default-expression-evaluation.